mock/src/main/java/com/tngtech/keycloakmock/impl/handler/LogoutRoute.java

```diff
@@ -14,10 +14,20 @@
 import javax.inject.Inject;
 import javax.inject.Singleton;
 
+/**
+ * @see <a
+ *     href="https://github.com/keycloak/keycloak-documentation/blob/main/securing_apps/topics/oidc/java/logout.adoc">Logout</a>
```

Comment:
I usually link to the published documentation, e.g.
```suggestion
 *     href="https://www.keycloak.org/docs/latest/securing_apps/index.html#logout">Logout</a>
```